Vehicle.js

```diff
@@ -0,0 +1,8 @@
+export class Vehicle {

```

Comment:
Poți face 

```js
export default class Vehicle {
```

Apoi, când imporți, faci doar

```js
import Vehicle from "./Vehicle.js"
```

Dar să știi că nu e greșit nici cum ai făcut tu. Unii chiar preferă să facă așa.